Makefile

```
@@ -46,36 +48,52 @@ test: test-unit test-integration
 test-unit:
 	go test -failfast -race -short -v ./...
 
+test-unit-memdb:
+	go test -failfast -race -tags memdb -short -v ./...
+
 test-unit-postgres:
 	go test -failfast -race -tags postgres -short -v ./...
 
 test-unit-cover:
 	go test -failfast -short -v -coverprofile=coverage.txt -covermode=count -coverpkg=all $(go list ./... | grep -v /demo/)
 
+test-unit-memdb-cover:
+	go test -failfast -short -tags memdb -v -coverprofile=coverage-memdb.txt -covermode=count -coverpkg=all $(go list ./... | grep -v /demo/)
+
 test-unit-postgres-cover:
-	go test -failfast -short -tags integration -v -coverprofile=coverage-postgres.txt -covermode=count -coverpkg=all $(go list ./... | grep -v /demo/)
+	go test -failfast -short -tags postgres -v -coverprofile=coverage-postgres.txt -covermode=count -coverpkg=all $(go list ./... | grep -v /demo/)
 
 test-integration:
 	go test -failfast -v ./demo
 	cd demo && go build && ./demo -build -test -debug
 
+test-integration-memdb:
+	go test -failfast -race -short -tags memdb -v ./...
+	cd demo && go build && ./demo -dbtype=memdb -build -test -debug
+
 test-integration-postgres:
-	go test -failfast -v ./demo
-	go test -failfast -race -short -tags integration -v ./...
+	go test -failfast -race -short -tags postgres -v ./...
 	cd demo && go build && ./demo -dbtype=postgres -build -test -debug
 
 coverage:
 	go get -v -t -d ./...
 	go test -failfast -v -covermode=atomic -coverpkg ./... -coverprofile=coverage.txt ./...
 
+coverage-memdb:
+	go get -tags=memdb -v -t -d ./...
+	go test -failfast -race -v -tags=memdb -covermode=atomic -coverpkg ./... -coverprofile=coverage-memdb.txt ./...
```

```suggestion
	go test -failfast -v -tags=memdb -covermode=atomic -coverpkg ./... -coverprofile=coverage-memdb.txt ./...
```
Coverage is slow enough as is.